app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -53,15 +53,13 @@ environment
 
 ## Your classroom placements
 
-Typically, you will get 120 days of classroom experience in a minimum of two schools. Through your placements, you’ll have the opportunity to teach, plan, observe and research.
+You should get 120 days of classroom experience as part of a postgraduate or 3 year undergraduate teacher training course. On a 4 year undergraduate course, you should get 160 days of classroom experience. The placements will be in a minimum of 2 schools. Through your placements, you’ll have the opportunity to teach, plan, observe and research.
```

Comment:
this is quite a big block of text (esp on mobile) - should we break it up a bit?